src/js/controller/controller.js

```diff
@@ -298,7 +298,7 @@ Object.assign(Controller.prototype, {
 
             // Autostart immediately if we're not waiting for the player to become viewable first.
             if (_model.get('autostart') === true && !_model.get('playOnViewable')) {
-                _autoStart();
+                _autoStart({ reason: 'autostart' });
```

Comment:
External `play` and `pause` methods need `meta` with `reason`. This call only needs the `reason` string:

`_autoStart('autostart');`